libs/database/src/lib/services/authentication.service.v2.ts

```diff
@@ -135,10 +135,10 @@ export class AuthenticationServiceV2 {
     return {
       accessToken,
       refreshToken,
-      profile: await user.toUserResponseV2({
+      profile: await user?.toUserResponseV2({
         passwordNotSet: !account.password,
       }),
-      pages: pages.map((page) => page.toPageResponseV2()),
+      pages: user && pages.map((page) => page.toPageResponseV2()),
```

Comment:
กรณีไม่มี user ไม่ควร return false ออกไปนะ ควรเป็น null หรือ undefined หรือ empty array จะ make sense กว่านะครับ

```suggestion
      pages: pages.map((page) => page.toPageResponseV2()),
```

Reply:
แบบนี้ มัน return undefined นะครับ ไม่ใช่ boolean
= user ? pages.map((page) => page.toPageResponseV2() : undefined